repo-s3-mirror

```diff
@@ -1,26 +1,28 @@
-#!/usr/bin/env python
-# Script to create a mirror of alibuild repository on 
-# and S3 bucket
-BUCKET=${BUCKET:-alibuild-repo}
-ENDPOINT=${ENDPOINT:-s3.cern.ch}
+#!/bin/sh -x
+# Script to create a mirror of alibuild repository in an S3 bucket.
 
-# Helper which resolves a symbolink link and stores the file it point to in the
-# associated S3 object
-copy_link() {
-  DST=$1
-  readlink $DST | s3cmd put -s -v --host $ENDPOINT --host-bucket $BUCKET.ENDPOINT - s3://$BUCKET/$DST
-}
+. /secrets/aws_bot_secrets
+export AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY
+
+bucket=${BUCKET:-alibuild-repo}
+prefix=/build/reports/repo
+repo=TARS/slc7_x86-64
```

Comment:
You should probably make these configurable.

Reply:
@ktf, see #841